boot/freeldr/freeldr/include/arch/pc/x86common.h

```diff
@@ -12,11 +12,13 @@
 #define BIOSCALLBUFFER      HEX(4000) /* Buffer to store temporary data for any Int386() call */
 #define STACK16ADDR         HEX(6F00) /* The 16-bit stack top will be at 0000:6F00 */
 #define BSS_START           HEX(6F00)
-#define STACKLOW            HEX(7000)
+#define STACKLOW            HEX(8000)
 #define STACKADDR           HEX(F000) /* The 32/64-bit stack top will be at 0000:F000, or 0xF000 */
-#define FREELDR_BASE        HEX(F800)
-#define FREELDR_PE_BASE    HEX(10000)
-#define MEMORY_MARGIN      HEX(88000) /* We need this much memory */
+#define FREELDR_BASE        HEX(20000)
+#define FREELDR_PE_BASE     HEX(21000)
+#define TEMPCODE_BASE       HEX(200000)
+#define TEMPCODE16_BASE     HEX(7000)
+#define MEMORY_MARGIN       HEX(9FC00) /* We need this much memory */
```

Comment:
This will have to be changed back before the merge.

Reply:
Yes. I will revert this hack when it's ready for review.

Reply:
Warning: changing it back to `HEX(88000)` will limit freeldr to have `408KBs` but i can change it to `HEX(99000)` because `HEX(9FC00)` is too big.